syz-manager/manager.go

```diff
@@ -1007,16 +1016,28 @@ func (mgr *Manager) saveRepro(res *repro.Result, stats *repro.Stats, hub bool) {
 		// 2. Repro re-tried 3 times and still got corrupted report at the end,
 		//    so maybe corrupted report detection is broken.
 		// 3. Reproduction is expensive so it's good to persist the result.
+
+		report := repro.Report
+		output := report.Output
+		if res.strace != nil && res.strace.Report != nil &&
```

Comment:
Re `res.strace.Report != nil`, some bugs don't don't contain report, but I see no reason to not provide strace output for them.

Reply:
In this case `res.strace.Report` is an object of type `*report.Report`, so it will be non-nil for all bugs.

> And for datastore entity as well.

What do you mean?

Reply:
>> And for datastore entity as well.
> What do you mean?

I meant this comment for dashapi.go regarding CrashLogFlags field.